shared/test/frontend_tester.py

```diff
@@ -17,14 +17,59 @@
 # need to consider whether this code wants to be shared with WebDriver tests
 # for other browsers, though.
 #
+
+gCommonDir = None
```

Comment:
And, really, what it represents (i.e. common between what and what other thing?).